<launch>

    <arg name="targetIp1" default="10.60.23.136" />
    <arg name="targetPort1" default="14550" />
    <arg name="srcPort" default="14551" />

    <node name="dji2mav_bringup" pkg="dji_sdk_dji2mav" type="dji2mav_bringup" output="screen">
        <param name="targetIp1" type="string" value="$(arg targetIp1)" />
        <param name="targetPort1" type="int" value="$(arg targetPort1)" />
        <param name="srcPort" type="int" value="$(arg srcPort)" />
    </node>

</launch>
